lib/query/utils.js

```diff
@@ -26,6 +26,21 @@ const queryType = query => {
     return 'describe';
   }
 
+  if (
```

Comment:
Unless you're stripping out prefix declarations at the beginning of the query, you're not always going to hit one of these.

Example:

```
prefix foo: <http://messing.up.your/logic#>
insert data { foo:foo foo:trololo 42 }
```